### ISSUE TITLE
Support Azul Mission Control

### ISSUE THREAD

**Issue by jameswilliams1** [ajshastri/chocolatey-packages#3]
Thanks for the great work on these packages, they're a real life saver!

It would be very useful to support [Azul Mission Control](https://www.azul.com/products/components/azul-mission-control/) as a package, given the Zulu JDK distributions are already supported. If using one of the JDKs it's generally advisable to use Azul's Mission Control rather than the Oracle one (as it also works with Zulu 8 whereas the Oracle one doesn't).

I'm not 100% how the automation works here, but happy to help with this if you can provide some guidance.

**Comment by ajshastri** [comment]
It seems simple enough, I can do it just like LibericaMissionControl and JMC packages. It'll end up in the toolsDir without any java. Does that work?

The new ZMC requires Azul Zulu JDK 17 and above, so that would be something you will have to add manually.

**Comment by jameswilliams1** [comment]
> It seems simple enough, I can do it just like LibericaMissionControl and JMC packages. It'll end up in the toolsDir without any java. Does that work?
> 
> The new ZMC requires Azul Zulu JDK 17 and above, so that would be something you will have to add manually.

Yeah I think that's reasonable, anyone using it would have Zulu installed anyway.
